ocdata/analysis.py

```diff
@@ -0,0 +1,66 @@
+"""
+This submodule contains the scripts for VASP calculations
+post-analysis:
+
+(1) Bader Charges Analysis
+(2) pDOS Analysis (TODO)
+"""
+
+__author__ = 'Javier Heras-Domingo'
+__email__ = 'jherasdo@andrew.cmu.edu'
+
+import os
+import sys
+import pickle
+from shutil import which
+
+import warnings
+warnings.filterwarnings("ignore")
+
+
+
+#Run Bader Analysis after vasp calculation
+def run_bader(outdir='./'):
+	"""
```

Comment:
It looks like your text editor uses tabs. [It's better practice to uses 4 spaces instead of a tab.](https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces)